.github/matchers/unbehead-problem-matcher.json

```diff
@@ -0,0 +1,14 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "unbehead",
+      "pattern": [
+        {
+          "regexp": "^((?:Missing|Incorrect) header) in (.+)$",
```

Comment:
Can you confirm that the matcher works (just temporarily add a commit w/ broken and missing headers in some files). Faster than me testing this in the Indico codebase...